### ISSUE TITLE
Upload files blocked by CORS policy after changing default port to 8443

### ISSUE THREAD

**Issue by jingerpie** [kasmtech/workspaces-issues#197]
I modified port from a working single-sever installation according to: https://kasmweb.com/docs/latest/how_to/port_change.html

Below are what I discover:
1. CORS policy shows up after modifying the default port 443 to any other port
2. Use both sever IP and domain name to access session, both are not able to upload
3. Nginx Reverse proxy is setup correctly with CORS policy included.

```
Access to XMLHttpRequest at 'https://example.com:8443/desktop/0c0bca01-9ebd-445a-94e7-22be875033d5/uploads/upload' from origin 'https://example.com' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.

react-dropzone.js:1          POST https://example.com:8443/desktop/0c0bca01-9ebd-445a-94e7-22be875033d5/uploads/upload
```

**Comment by j-travis** [comment]
Hello, 
Can you try the following:

1. In the Zone settings, please change the proxy port in the Zone settings to the port you are using on that reverse proxy (e.g 443)
2. Remove any CORS settings you have in your reverse proxy
3. Create a new Kasm session. (Do not use an existing session)

**Comment by jingerpie** [comment]
I have tried the method you provided. If I change the port to 443, it will resolve the problem, but once I change back to 8443, the same issue will show up.

**Comment by teja-kasm** [comment]
We’re currently enhancing our GitHub issue tracker to improve organization and efficiency.

As part of this process, we’re closing older issues to help keep the tracker focused on current and active topics.

If this issue is still relevant or you’re still experiencing the problem, please feel free to create a new issue.

Thank you for your understanding!
